### ISSUE TITLE
Accessibility app translation files are missing

### ISSUE THREAD

**Issue by mjanssens** [nextcloud/server#11803]
Noticed in 14.0.3 translation for the Accessibility app isn't there.
Looking in server/apps/accessibility/ the l10n folder isn't there so translation support has yet to be added.

**Comment by JSoko** [comment]
Still persists in 15.0.0...
